src/routes/delegates/delegates.controller.ts

```diff
@@ -67,4 +77,19 @@ export class DelegatesController {
   ): Promise<unknown> {
     return this.service.postDelegates(chainId, createDelegateDto);
   }
+
+  @ApiCreatedResponse()
```

Comment:
Why do you need to annotate with this decorator?

Reply:
Was a copy paste error from post :man_facepalming: fixed [here](https://github.com/5afe/safe-client-gateway-nest/pull/146/commits/0b1bb4165478747d8ffa85b5f9c57b07d9c6a657)